utilities/src/biginteger/biginteger.rs

```diff
@@ -100,6 +100,12 @@ pub trait BigInteger:
     /// leading zeros.
     fn to_bits(&self) -> Vec<bool>;
```

Comment:
Can you rename all usages of this to `to_bits_be` and update the comment to be more descriptive?